be/src/storage/lake/tablet_manager.cpp

```diff
@@ -2,58 +2,143 @@
+    return fmt::format("{}/tbl_{:016X}_{:016X}", group, tablet_id, verson);
+}
+
+// trnslog path rule $Bucket:/$ServiceID/$GroupID/txn_${TabletID}_${TxnID}
+std::string TabletManager::txn_log_path(const std::string& group, int64_t tablet_id, int64_t txn_id) {
+    return fmt::format("{}/txn_{:016X}_{:016X}", group, tablet_id, txn_id);
+}
+
 Status TabletManager::create_tablet(const TCreateTabletReq& req) {
-    (void)req;
-    return Status::NotSupported("TabletManager::create_tablet");
+    // generate metapb
+    TabletMetadataPB tablet_metadata_pb;
+    tablet_metadata_pb.set_id(req.tablet_id);
+    tablet_metadata_pb.set_version(1);
+
+    // rowset metadata with version 1
+    auto rowset_metadata_pb = tablet_metadata_pb.add_rowsets();
+    rowset_metadata_pb->set_id(1);
+    rowset_metadata_pb->set_overlapped(false);
+    rowset_metadata_pb->set_data_size(0);
+    rowset_metadata_pb->set_num_rows(0);
+    tablet_metadata_pb.set_next_rowset_id(2);
+
+    // get shard group
+    ASSIGN_OR_RETURN(auto group_path, get_staros_shard_path(req.tablet_id));
+
+    // write tablet meta
+    auto meta_path = tablet_meta_path(group_path, tablet_metadata_pb.id(), tablet_metadata_pb.version());
+    ASSIGN_OR_RETURN(auto fs, FileSystem::CreateSharedFromString(meta_path));
+    ASSIGN_OR_RETURN(auto wf, fs->new_writable_file(meta_path));
+    RETURN_IF_ERROR(wf->append(tablet_metadata_pb.SerializeAsString()));
+    RETURN_IF_ERROR(wf->close());
+    return Status::OK();
 }
 
 StatusOr<Tablet> TabletManager::get_tablet(int64_t tablet_id) {
-    (void)tablet_id;
-    return Status::NotSupported("TabletManager::get_tablet");
+    ASSIGN_OR_RETURN(auto group_path, get_staros_shard_path(tablet_id));
+    return Tablet(this, group_path, tablet_id);
 }
 
 Status TabletManager::drop_tablet(int64_t tablet_id) {
-    (void)tablet_id;
-    return Status::NotSupported("TabletManager::drop_tablet");
+    vector<std::string> objects;
+    // get shard group
+    ASSIGN_OR_RETURN(auto group_path, get_staros_shard_path(tablet_id));
+    ASSIGN_OR_RETURN(auto fs, FileSystem::CreateSharedFromString(group_path));
+
+    auto scan_cb = [&objects, &tablet_id](std::string_view name) {
+        bool ret = false;
+        std::string id_str;
+        std::string version_str;
+        static re2::RE2 re(R"(txn_([A-Fa-f0-9]+)_([A-Fa-f0-9]+)|tbl_([A-Fa-f0-9]+)_([A-Fa-f0-9]+))");
+        ret = RE2::PartialMatch(std::string(name), re, &id_str,&version_str);
+        if (!ret) {
+            // parsed tableid failed, just skip it ;
+            return true;
+        }
+        StringParser::ParseResult result;
+        auto parsed_tabletid = StringParser::string_to_int<int64_t>(id_str.data(), id_str.size(), &result);
+        if (result != StringParser::PARSE_SUCCESS || parsed_tabletid != tablet_id) {
+            // skip object for other tablet
+            return true;
+        }
```

Comment:
Only need to check if `name` was started with `tbl_${tablet_id}` or `txn_${tablet_id}`